classes/class-exoskeleton.php

```diff
@@ -292,7 +292,14 @@ private function back_off_please( $rule_id, $matched_rule, $lock ) {
 		@header( "Cache-Control: public max-age=$retry_after" );
 		status_header( 429, 'Exoskeleton: too many requests for this endpoint.  Please consult Retry-After and come back later.  Meanwhile enjoy a well-deserved REST' );
 		@header( "Retry-After: $retry_after" );
+
+		// Do not die if testing.
+		if ( defined( 'PHPUNIT_EXOSKELETON_TESTING' ) ) {
```

Comment:
Does it make sense to calculate the retry after headers in a different function that would be easier to test? Then we could just output the headers and exit here without altering any logic for tests.